srcpkgs/minipro/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'minipro'
+pkgname=minipro
+version=0.5
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
+hostmakedepends="pkg-config libusb-devel"
+makedepends="libusb-devel which"
+depends="libusb"
+short_desc="Program for controlling the MiniPRO TL866xx series of chip programmers"
+maintainer="jleightcap <jleightcap@protonmail.com>"
+license="GPL-3.0-only"
+homepage="https://gitlab.com/natehouk/minipro"
+distfiles="https://gitlab.com/natehouk/${pkgname}/-/archive/${version}/${pkgname}-${version}.tar.gz"
```

Comment:
If you're going to expand in distfiles, since homepage is a parent, better to use it:
`distfiles="${homepage}/-/archive/${version}/${pkgname}-${version}.tar.gz"`